client/src/components/card/effect-card-svg.tsx

```diff
@@ -48,6 +49,16 @@ const HermitCard = ({card}: EffectCardProps) => {
 					EFFECT
 				</text>
 			</g>
+			{['rare', 'ultra_rare'].includes(card.rarity) ? (
```

Comment:
This will now appear even in the game. Honestly I feel like the stars don't look that nice on effects/item so I'd prefer to use them only on the deck building screen.

Reply:
thats why i had console log to check if the title is different so i can only show stars on deck screen
do you know any specific thing that says you are in deck building screen??